ClLike/cl_like/cl_like.py

```diff
@@ -121,9 +118,14 @@ def _init_globals(self):
             self.p2pt_HOD = ccl.halos.Profile2ptHOD()
             # Halo model correction for the transition regime
             if self.HM_correction == 'halofit':
+                from .hm_extra import HalomodCorrection
                 self.hmcorr = HalomodCorrection()
             else:
                 self.hmcorr = None
+        elif self.bias_model == 'LagrangianPT':
+            from .lpt import LPTCalculator, get_lpt_pk2d
```

Comment:
I don't think this works. `LPTCalculator` and `get_lpt_pk2d` won't be visible outside of `__init_globals`, right? Same for the others.

Reply:
You're absolutely right. Any ideas? The other options that I have now in mind are:
- To move the imports to the functions where the functions are needed, but I find that a bit ugly
- Hack it by making those variables global. Also ugly